src/python/WMCore/Services/WMArchiver/DataMap.py

```diff
@@ -247,6 +252,8 @@ def createArchiverDoc(job_id, fwjr, version=None):
     newfwjr['meta_data'] = {'agent_ver': version,
                          'host': socket.gethostname().lower(),
                          'fwjr_id': job_id,
-                         'ts': int(time.time())
+                         'jobtype': jobtype,
```

Comment:
Yes, I will update new schema after running the test. With this patch I can extract all different kinds of fwjr by type and its status. I will change necessary schema changes and create the new pull request in WMArchive.